src/character.cpp

```diff
@@ -4057,16 +4067,22 @@ void Character::set_stored_kcal( int kcal )
 void Character::set_stored_calories( int cal )
 {
     if( stored_calories != cal ) {
+        int cached_bmi = std::floor( get_bmi_fat() );
         stored_calories = cal;
 
         //some mutant change their max_hp according to their bmi
         recalc_hp();
+        //need to check obesity penalties when this happens if BMI changed
+        if( std::floor( get_bmi_fat() ) != cached_bmi ) {
+            calc_encumbrance();
+        }
     }
 }
 
 int Character::get_healthy_kcal() const
 {
-    return healthy_calories / 1000;
+    float healthy_weight = 5.0f * std::pow( height() / 100.0f, 2 );
+    return std::floor( 7716.17 * healthy_weight );
```

Comment:
What is the 7716.17 here? It seems like it should be a const to avoid having a magic number.

Reply:
7716.17 is 3500 * 2.20462 which is the number of kilocalories in a kilogram of fat. It could potentially be defined somewhere else, just not sure how to do that exactly.

Reply:
Easiest way would be to make it a const in character.h make sure to leave a comment above it explaining what it is.